docs/settings/ingest-manager-settings.asciidoc

```diff
@@ -0,0 +1,56 @@
+[role="xpack"]
+[[ingest-manager-settings-kb]]
+=== Ingest Manager settings in Kibana
+++++
+<titleabbrev>Ingest Manager settings</titleabbrev>
+++++
+
+You can configure `xpack.ingestManager` settings in your `kibana.yml`
+
+[[general-ingest-manager-settings-kb]]
+==== General Ingest Manager settings
+
+[cols="2*<"]
+|===
+| `xpack.ingestManager.enabled`
+  | Set to `true` to enable {kib} Ingest Manager. 
+|===
+
+[cols="2*<"]
+|===
+| `xpack.ingestManager.epm.enabled`
+  | Set to `true` (default) to enable {kib} EPM. 
```

Comment:
Avoid abbreviations like EPM, especially for new technology. I think we calling this simple "package manager" (see https://github.com/elastic/style-guide/issues/82#issuecomment-637005667).

Since product names are volatile, I will created some shared attributes that you can use to resolve these names.

Reply:
Update: I've opened a PR that contains the attributes. Just need to get the PM to review. If that PR isn't merged before you're ready, I can do a follow-up PR. Just let me know. https://github.com/elastic/docs/pull/1862

Reply:
Perfect I will make the change or you will depending on which PR is merged before